Maquetado/scss/login.scss

```diff
@@ -0,0 +1,71 @@
+@mixin login-item($margin) {
+  @extend %login-text;
+  margin: $margin;
+  width: 100%;
+}
+
+%login-text {
+  border-radius: 5px;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.login-box {
+  align-items: center;
+  background-color: $wild-sand;
+  display: flex;
+  flex-direction: column;
+  height: 400px;
```

Comment:
`max-height: 400px;`

Reply:
I think that with `height` is enough or not?

Reply:
Yes, it's true.
@martinmona what I told you yesterday apply only for `width` property, not for height. I'm sorry :p